library/alloc/src/collections/vec_deque/mod.rs

```diff
@@ -471,13 +478,45 @@ impl<T> VecDeque<T> {
     ///
     /// let vector: VecDeque<u32> = VecDeque::with_capacity(10);
     /// ```
+    #[inline]
     #[stable(feature = "rust1", since = "1.0.0")]
     pub fn with_capacity(capacity: usize) -> VecDeque<T> {
+        Self::with_capacity_in(capacity, Global)
+    }
+}
+
+impl<T, A: Allocator> VecDeque<T, A> {
+    /// Creates an empty `VecDeque`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::collections::VecDeque;
+    ///
+    /// let vector: VecDeque<u32> = VecDeque::new();
+    /// ```
+    #[inline]
+    #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
This looks off - there wasn't a `new_in` function before, was there? It looks like an implementation detail, in which case it should be private; or it's part of the allocator API, in which case it should be unstable with a tracking issue.

Reply:
It's a mistake, I marked it as `#[unstable]`